update articles link between assets & products (#431)
akeneo/pim-helpcenter: content/md/what-is/what-is-an-attribute.md

- **Identifier** attribute: a code to identify your product, **this code must be unique**. It can be a SKU, a MPN... **This attribute is mandatory to create products.**
- **Text** attribute: a single-line text field that can contain **up to 255 characters**, it is usually used for a product name.
- **Text area** attribute: a multi-line text field that can be used for a product description.
- **Simple select** attribute: a single-choice list coming with custom options. Only one value can be selected amongst the available options.
- **Multi select** attribute: a multi-choice list coming with custom options. More than one value can be selected amongst the available options.
- **Yes/No** attribute: a boolean attribute
- **Date** attribute: a date field, the PIM will display a calendar to choose the date, which includes day, month and year.
- **Number** attribute: a single-line field that can only contain digits.

> - **Metric** attribute: a single-line field composed of a 1st field containing a number and a second field containing a metric unit. It allows you to automatically convert metric values into others to fit your export needs. You can check the [Manage your Channels](manage-your-channels.html#create-a-channel) article to have more information about this feature.

Reviewer: ```suggestion
- **Metric** attribute: a single-line field composed of a first field containing a number and a second field containing a metric unit. It allows you to automatically convert metric values into others to fit your export needs. You can check the [Manage your Channels](manage-your-channels.html#create-a-channel) article to get more information about this feature.
```

:::

:::tips
To store GTIN, EAN, UPC… in your PIM, use the **text** attribute type and use a regular expression to control the field input.
For instance, to store an EAN13 in Akeneo which does always contain 13 digits, add a validation rule based on the following regular expression: `/^[0-9]{13}$/`
:::

## Reference entities or reference data?

### Differences between the two attribute types

> **Reference entities are an Enterprise Edition feature added in the 3.0 version**. To learn more about Reference entities, read our dedicated article [What is a Reference Entity?](what-about-reference-entities.html).    

Reviewer: ```suggestion
**Reference Entities is an Enterprise Edition feature added in the 3.0 version**. To learn more about Reference Entities, read our dedicated article [What is a Reference Entity?](what-about-reference-entities.html).    
```

> Whereas reference data are a native Community and Enterprise editions feature that manages links with products. They imply a custom development (installation of the [Custom Entity Bundle](https://github.com/akeneo-labs/CustomEntityBundle)) to enrich the reference data through the user interface, collect and distribute it.

Reviewer: ```suggestion
Reference data is a native Community and Enterprise Edition feature that manages links with products. They imply a custom development (installation of the [Custom Entity Bundle](https://github.com/akeneo-labs/CustomEntityBundle)) to enrich the reference data through the user interface, collect and distribute it.
```

### How can I import/export Reference entities?

> There is no native import/export for Reference entities, but you can use our API to manage them. Learn about their API format and the endpoints available on our API online documentation:

Reviewer: ```suggestion
There is no native import/export for Reference Entities, but you can use our API to manage them. Learn about their API format and the endpoints available on our API online documentation:
```
